uaclient/files.py

```diff
@@ -350,6 +352,147 @@ def contract_id(self):
         return None
 
 
+class NoticesLabelID(NoticeFileDetails, Enum):
```

Comment:
I like the idea of the notices as values of an enum. I think we might as well go all the way and include the notice message content in the tuple as well.

And if we do that we can rename the Enum to just `Notices`

Reply:
Some message content uses formatting tmpl which makes it  impossible to edit the enum since it is readonly.